rest-api-spec/src/main/resources/rest-api-spec/api/reindex.json

```diff
@@ -30,6 +30,11 @@
           "default": 0,
           "description": "The throttle to set on this request in sub-requests per second. -1 means no throttle."
         },
+        "scroll": {
+          "type" : "time",
+          "description" : "Control how long to keep the search context alive",
+          "default" : "5m"
```

Comment:
where did you find the 5m default again?

Reply:
> Both work exactly how they work in the Bulk API. As _reindex uses scroll search, you can also specify the scroll parameter to control how long it keeps the "search context" alive, (e.g. ?scroll=10m). The default value is 5 minutes.

https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html